src/constants/functions.js

```diff
@@ -0,0 +1,15 @@
+import moment from "moment";
```

Comment:
rename a file with dateFunctions.js. +1 for creating this file. Now the advantage of this would be if you change the moment to luxon you don't need to change the implementation inside your components because you have encapsulated this logic in a function and your component only knows this function. Also you can unit test these functions

Reply:
renamed the file.